src/pyrokinetics/gk_code/gk_output.py

```diff
@@ -690,17 +690,10 @@ def _eigenvalues_from_fields(
 
     @staticmethod
     def _eigenfunctions_from_fields(fields: Fields, theta: ArrayLike) -> Eigenfunctions:
-        # field coords are (theta, kx, ky, time)
-        square_fields = np.zeros(fields.shape)
-        for field in fields.values():
-            square_fields += np.abs(field.magnitude) ** 2
-        field_amplitude = np.sqrt(np.trapz(square_fields, theta, axis=0)) / (2 * np.pi)
-        eigenfunctions = np.zeros(
-            (len(fields.coords),) + square_fields.shape, dtype=complex
-        )
+        eigenfunctions = np.zeros((len(fields.coords),) + fields.shape, dtype=complex)
+
         for ifield, field in enumerate(fields.values()):
-            eigenfunctions[ifield] = field.magnitude / field_amplitude
-        # TODO are these dims correct?
+            eigenfunctions[ifield] = field.magnitude
 
```

Comment:
This looks like an unrelated change?

Reply:
This seems to be from #280 but not sure why its come up here

Reply:
Probably merged it wrong